howto/install-appliance/landing.md

```diff
@@ -0,0 +1,14 @@
+The Anbox Cloud Appliance provides a deployment of Anbox Cloud to a single machine. This offering is well suited for initial prototype and small scale deployments.
+The guides in this section describe how to install the Anbox Cloud Appliance on a cloud platform.
+
+We strongly recommend that you follow the [Install the Anbox Cloud Appliance on your local machine](https://discourse.ubuntu.com/t/install-appliance/22681) tutorial before you install the appliance on a cloud platform. The tutorial guides you through installing the appliance on your local machine and makes you familiar with both the installation process and the general concepts of the Anbox Cloud Appliance.
+
+[note type="information" status="Note"]There is a difference between the full Anbox Cloud installation and the Anbox Cloud Appliance (see [Variants](https://discourse.ubuntu.com/t/anbox-cloud-overview/17802#variants)). This section focuses on the **Anbox Cloud Appliance**. For instructions on how to install **Anbox Cloud**, see [How to install Anbox Cloud](https://discourse.ubuntu.com/t/install-anbox-cloud/24336).[/note]
+
+Make sure to check the [Requirements](https://discourse.ubuntu.com/t/installation-requirements/17734) before you start your installation.
+
+The Anbox Cloud Appliance is currently available for the following cloud platforms:
+
+* [AWS](tbd)
+
+[Oracle Cloud (OCI)](https://www.oracle.com/cloud/) is also supported, but the Anbox Cloud Appliance is not available from the Oracle Cloud Marketplace yet. Therefore, to install the appliance on OCI, follow the [Install the Anbox Cloud Appliance on your local machine](https://discourse.ubuntu.com/t/install-appliance/22681) tutorial and install the Anbox Cloud Appliance snap on OCI.
```

Comment:
Can we make this more generic and say "any other" cloud (specifically Azure, OCI and Google Cloud)?